engli/kanun.md

```diff
@@ -891,6 +891,22 @@ With the modified noun at the beginning, we need to call into action the otherwi
 **yi kalam da su me zay kitaba si novi.**
 – This pen, with which I’m writing now, is new.
 
+#### 9.2.5. Modification with clauses
+
+Any clause can modify a noun in the same manner: the modified noun is moved to the beginning of the modifying clause
+and separated out with **da**, or moved to the end and separated out with **du**.
+
+**me xerca me paso abla laxa du eni xey.**
+– I'm looking for anything I might have left behind.
+
+In more complex cases, it can become challenging to maneuver the noun to the end of the clause. In these cases, the
+word order may be relaxed as long as it is clear where in the modifying clause the noun goes. Note, however, that it
+may be more understandable to simply rearrange the sentence entirely.
+
+**le loga loke da me no abla eureka sa karte.**
+– They named a place I couldn't find on the map.
```

Comment:
Do you intend to use they as a singular pronoun? We haven't done that before in examples. Maybe it's better to use lole in Pandunia version.

Reply:
That was my intention, but **lole** works just as well.